templates/servicemonitor.yaml

```diff
@@ -0,0 +1,20 @@
+{{ if .Values.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "docker-registry.fullname" . }}
+  labels:
+    {{- include "docker-registry.labels" . | nindent 4 }}
```

Comment:
:+1: I like this -- much cleaner than the hard-coded/duplicated labels in each of the other templates. Can this be extracted into a separate pull request, though? If this PR introduces a `docker-registry.labels` helper, I'd like to simultaneously adopt that helper consistently across all templates instead of applying to just one.

Reply:
Sounds reasonable - this was mainly part of my PR because I already used it in a helm chart I had written for the same purpose as this ( which I wrote because I thought this one had been abandoned). I can't claim credit for that though, it is a standard feature of modern helm charts when building them via `helm create`. 

I will extract it for the time being and create a separate PR based on on this one.